include/circt/Dialect/SV/SVPasses.h

```diff
@@ -39,8 +39,7 @@ std::unique_ptr<mlir::Pass>
 createSVExtractTestCodePass(bool disableInstanceExtraction = false,
                             bool disableRegisterExtraction = false,
                             bool disableModuleInlining = false);
-std::unique_ptr<mlir::Pass>
-createHWExportModuleHierarchyPass(std::optional<std::string> directory = {});
```

Comment:
I don't think we need to change the signature of the function in this PR, right? I would prefer to not change anything in this PR beyond moving the remaining passes into new Firtool.h functions.

Reply:
👍 thanks. It makes sense that it was unused, but I think we would prefer a separate small commit to clean that up, rather than changing the pass create API while also doing this PR.